framework/test/unit/components/storage/entities/round.js

```diff
@@ -678,7 +678,7 @@ describe('Round', () => {
 	});
 
 	describe('performRoundSnapshot()', () => {
-		afterEach(() => RoundEntity.clearRoundSnapshot());
+		afterEach(async () => RoundEntity.clearRoundSnapshot());
```

Comment:
There are more similar `afterEach` and also `beforeEach` than these that you modify. Why not adding `async` to all of them?

Reply:
I just focused on solving this problem. Please feel free to create an issue to make sure we are handling all Promises correctly.